### PR TITLE
Added `model.eval()` and `torch.no_grad()` to disable dropout/gradient calculation during inference

### PR DESCRIPTION
**Pull request by salman-moh**
Referencing #13 
Simply put, I was getting **different predictions** for the **same SMILE input**, turns out this additional resolved that issue.

### CODE REVIEW

**Comment by salman-moh** [comment]
Had to undo the other changes to put a PR for 1 change at a time.

**Comment by salman-moh** [comment]
I did not re-train the model as we changed our focus for now... but during
inference there was a +/- difference of 0.3-0.4 on the single ESOL
predictions. I'll report back once I get on this.
